contentcuration/contentcuration/frontend/channelList/router.js

```diff
@@ -12,24 +12,34 @@ import ChannelModal from 'shared/views/channel/ChannelModal';
 const router = new VueRouter({
   routes: [
     {
-      name: RouterNames.CHANNELS,
-      path: '/channels/:listType',
-      props: true,
+      name: RouterNames.CHANNELS_EDITABLE,
+      path: `/${ListTypes.EDITABLE}`,
       component: ChannelList,
-      children: [
-        {
-          name: RouterNames.CHANNEL_DETAILS,
-          path: ':channelId/details',
-          component: ChannelDetailsModal,
-          props: true,
-        },
-        {
-          name: RouterNames.CHANNEL_EDIT,
-          path: ':channelId/edit',
-          component: ChannelModal,
-          props: true,
-        },
-      ],
+      props: { listType: ListTypes.EDITABLE },
+    },
+    {
+      name: RouterNames.CHANNELS_STARRED,
+      path: `/${ListTypes.STARRED}`,
```

Comment:
Yes, that's a good point, I was thinking about removing the constants completely, too, just was a bit afraid because we have relatively lots of in-component routes dependencies in general and there was more logic around `ListTypes`. I realize now that lots of them have been removed though so maybe it will be fine. I will check it and if it's not way too much complicated, will do, otherwise open a follow-up.

Reply:
I've just removed those constants and looks good.